DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/IntitialMenuViewModelTests.cs

```diff
@@ -40,23 +41,23 @@ public void Initial_menu_should_have_id()
             var initialMenuViewModel = new InitialMenuViewModel(expectedCourseContent);
 
             // Then
-            initialMenuViewModel.Id.Should().Be(12);
+            initialMenuViewModel.Id.Should().Be(customisationId);
         }
 
         [Test]
         public void Initial_menu_should_have_averageDuration()
         {
             // Given
-            const int customisationId = 12;
+            const string averageDuration = "TestDuration";

```

Comment:
Very minor but I think it's best to set this to the format we expect it to be in. Otherwise it can be a bit confusing 🙂